src/components/who.js

```diff
@@ -4,74 +4,117 @@ import styled from "styled-components"
 import Section from "../styles/Section"
 import Header from "../styles/Header"
 import Img from "gatsby-image"
+import shovel from "../images/shovel.svg"
 
-// Probably a better name
-const ContainerContainer = styled.div`
-  display: flex;
+// SVG gets inlined as base64. This is ok because it's only 3 kb
+const Mobile = styled.div`
```

Comment:
Why `min-width` over `max-width` for queries? Is this for the mobile-first development philosophy?

Reply:
Resolved.